### PR TITLE
Bump v1.26

### PR DESCRIPTION
**Pull request by ingvagabund**
- bump k8s dependencies to v1.26
- wire logging flags into the NewCommandStartAdmissionServer flags

### CODE REVIEW

**Comment by ingvagabund** [comment]
/hold
until proven the admission server is running and serving

**Comment by deads2k** [comment]
/lgtm
/approve
/hold for proof


The log flag changes are fine.  klog continues to shift.

**Comment by ardaguclu** [comment]
It seems that this https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_run-once-duration-override/8/pull-ci-openshift-run-once-duration-override-main-e2e-aws-operator/1620788462888161280/artifacts/e2e-aws-operator/gather-extra/artifacts/pods/run-once-duration-override_runoncedurationoverride-gbpn5_runoncedurationoverride.log proves that admission server is running and serving

**Comment by ingvagabund** [comment]
Yes!!! We can see the admission.go logs:
```
I0201 14:53:42.179219       1 admission.go:125] namespace=e2e-testmutation - admitting resource
I0201 14:53:42.179425       1 admission.go:142] namespace=e2e-testmutation pod activeDeadlineSeconds after mutating is: 0xc000a7ca98
```

File: pkg/cmd/cmd.go [hunk 1]
```diff
@@ -36,9 +30,7 @@ func RunAdmissionServer(admissionHooks ...AdmissionHook) {
 	for i := range admissionHooks {
 		castSlice = append(castSlice, admissionHooks[i])
 	}
-	cmd := server.NewCommandStartAdmissionServer(os.Stdout, os.Stderr, stopCh, castSlice...)
-	cmd.Flags().AddGoFlagSet(flag.CommandLine)
-	if err := cmd.Execute(); err != nil {
-		klog.Fatal(err)
-	}
+
+	code := cli.Run(server.NewCommandStartAdmissionServer(os.Stdout, os.Stderr, stopCh, castSlice...))
```
Comment (deads2k) [inline pkg/cmd/cmd.go:34]: that's nice.

**Pull request merged**